Userland/Utilities/unzip.cpp

```diff
@@ -111,6 +114,20 @@ int main(int argc, char** argv)
         return 1;
     }
 
+    if (!output_directory_path.is_empty()) {
```

Comment:
Let's ensure we don't allow `-o ""`, just use the string's null state instead meaning the argument wasn't passed:

```suggestion
    if (!output_directory_path.is_null()) {
```